src/database/DataTypes/Item.js

```diff
@@ -148,7 +148,7 @@ export class Item extends Realm.Object {
    * @return {Number} this items monthly usage based on a 30 day month.
```

Comment:
:nit - the comment could be removed

Reply:
e4a3649a992c583cc1c5bab8258273515d1eef74